xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -110,4 +114,5 @@ class CProcessInfo
   // player states
   CCriticalSection m_stateSection;
   bool m_stateSeeking;
+  std::atomic_int m_levelVQ;
```

Comment:
RP must not use VP's private APIs. I told you many times already. VP takes care that VQ represents actual state of MQ